request.proto

```diff
@@ -78,6 +78,9 @@ message StreetNetworkParams {
     optional int32 max_bss_duration_to_pt       = 15;
     optional int32 max_car_duration_to_pt       = 16;
     optional bool enable_direct_path            = 17 [default=true];
+
+    optional double ridesharing_speed           = 18;
+    optional int32 max_ridesharing_duration_to_pt = 19;
```

Comment:
not fan of the naming : `car_no_park` if we have a new mode, maybe a bool `with_park` if possible...